lib/app/modules/onc_program/onc_ehr_launch_sequence.rb

```diff
@@ -17,63 +17,69 @@ class OncEHRLaunchSequence < EHRLaunchSequence
 
       defines :token, :id_token, :refresh_token, :patient_id
 
-      @@resource_types = [
-        'Patient',
-        'AllergyIntolerance',
-        'CarePlan',
-        'Condition',
-        'Device',
-        'DiagnosticReport',
-        'DocumentReference',
-        'Encounter',
-        'ExplanationOfBenefit',
-        'Goal',
-        'Immunization',
-        'Medication',
-        'MedicationDispense',
-        'MedicationStatement',
-        'MedicationOrder',
-        'Observation',
-        'Procedure',
-        'DocumentReference',
-        'Provenance'
-      ]
-
-      test 'Scopes enabling user-level access with OpenID Connect and Refresh Token present' do
+      def valid_resource_types
```

Comment:
Should this include USCoreR4 resources like CareTeam, Location, MedicationRequest, Organization, Practitioner, and PractitionerRole?

Reply:
Are we waiting on something here?

Reply:
I was just waiting on a resolution to the other issues before I addressed this.

Reply:
Now the US Core IG says that servers should support ValueSet $expand. Is that a read scope?